npm/npm_import.bzl

```diff
@@ -430,6 +430,12 @@ WARNING: `package_json` attribute in `npm_translate_lock(name = "{name}")` is de
         if "no-sandbox" not in lifecycle_hooks_execution_requirements["*"]:
             lifecycle_hooks_execution_requirements["*"].append("no-sandbox")
 
+    # TODO (2.0): Stop defaulting patch_args with {"*": ["-p0"]} when no patches are supplied
+    # as this conflicts with the default pnpm.patchedDependencies format of "-p1" and
+    # needs to be explicitly overridden by the user.
+    if len(patches) == 0:
+        patch_args = {}
+
```

Comment:
I figure this isn't a breaking change because it won't affect anyone who doesn't have `patches`.